File: JAVA/Gpu.java
public class Gpu extends Komponen {
    private int vramGB;
    private String tipeMemory;

    public Gpu() {}

    public Gpu(int vramGB, String tipeMemory, String merk, String nama) {
        super(merk, nama);
        this.vramGB = vramGB;
        this.tipeMemory = tipeMemory;
    }

    public void setVramGB(int vramGB) {
        this.vramGB = vramGB;
    }

    public void setTipeMemory(String tipeMemory) {
        this.tipeMemory = tipeMemory;
    }

    public int getVramGB() {
        return vramGB;
    }

    public String getTipeMemory() {
        return tipeMemory;
    }
}

File: PYTHON/Gpu.py
from Komponen import Komponen

class Gpu(Komponen):
    def __init__(self, vramGB=0, tipeMemory="", merk="", nama=""):
        super().__init__(merk, nama)
        self.vramGB = vramGB
        self.tipeMemory = tipeMemory

    def setVramGB(self, vramGB):
        self.vramGB = vramGB

    def setTipeMemory(self, tipeMemory):
        self.tipeMemory = tipeMemory

    def getVramGB(self):
        return self.vramGB

    def getTipeMemory(self):
        return self.tipeMemory
